src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -218,7 +218,7 @@ public string Version
         /// <summary>
         /// Host analytics info
         /// </summary>
-        public HostAnalyticsInfo HostAnalyticsInfo { get; set; }
+        public static HostAnalyticsInfo HostAnalyticsInfo { get; set; }
```

Comment:
This is a good idea and I have some other change would benefit from this

Reply:
There are still some places where HostAnalytics is not static (ex. SplashScreen.xaml.cs, PackageManagerExtension.cs, IStartConfiguration and DefaultStartConfiguration, ReadyParams, StartDynamoWithDefaultConfig)
Do they all still make sense to exist and to be non-static ?
Some of these changes will conflict with the "clean obsolete APIs" PR
FYI @sm6srw @mjkkirschner

Reply:
Yeah, raised this concern as well above -thanks for pointing out how many non static usage cases still exist.
In addition
@zeusongit's PR touches this as well dealing with the fact that Application.Current is no longer null in Revit.

Reply:
Imo the start configurations data structures should remain as is. Those are transient and host info from there ends up being set in DynamoModel on startup.
ReadyParams hostinfo ( perhaps we should just remove it ?! ) delegates towards DynamoModel static now.
Handling host info in SplashScreen.xaml.cs and PackageManagerExtension.cs is a little bit above the scope of this PR so let's handle that separately in the other or a different pr. The model not being always available I believe is the reason why we have separate instances of host info in splash screen and packagemanager.